src/plugins/add-cal/reports/a11y-1.json

```diff
@@ -0,0 +1,900 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2022/acr-new-wcag-sc.json-ld",
```

Comment:
This context are only for newly Success Criterio added by WCAG 2.1 and 2.2 not for the one that was already existing in WCAG 2.0.

We will need to create a new JSON-LD context that would support all SC in WCAG 2.2.

Reply:
Thanks for the PR in the vocab repository. I am going to create a new context with the new term we added recently